AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -11,7 +11,10 @@ protocol MeasurementStreamStorage {
     func createSession(_ session: Session) throws
     func createSessionAndMeasurementStream(_ session: Session, _ stream: MeasurementStream) throws -> MeasurementStreamLocalID
     func updateSessionStatus(_ sessionStatus: SessionStatus, for sessionUUID: SessionUUID) throws
+    func updateSessionEndtime(_ endTime: Date, for sessionUUID: SessionUUID) throws
     func updateSessionFollowing(_ sessionStatus: SessionFollowing, for sessionUUID: SessionUUID)
+    
+    func existingMeasurementStream(_ sessionUUID: SessionUUID, name: String) throws -> MeasurementStreamLocalID?
```

Comment:
Same throwing thing

Reply:
This is a throwing func bc it's checking if the session exists and this func can throw `context.existingSession(uuid: sessionUUID)`. Not sure if I'm answering your question in understandable manner :P